index.ts

```diff
@@ -164,7 +164,9 @@ const transcriptionEventHandler = async (event: AudioBytesEvent) => {
   if (!transcriptionMutex && joinedBuffer.length > ONE_SECOND) {
     transcriptionMutex = true;
     globalWhisperPromise = whisper.whisperInferenceOnBytes(joinedBuffer);
-    const transcription = await globalWhisperPromise;
+    const rawTranscription = await globalWhisperPromise;
+    // Remove transcription artifacts like (wind howling)
+    const transcription = rawTranscription.replace(/\s*\[[^\]]*\]\s*|\s*\([^)]*\)\s*/g, '');
```

Comment:
You can also add a check that if transcription is an empty string after this, just don't do anything. Right now we will still fire the transcriptions event